src/parser.ts

```diff
@@ -1200,11 +1200,42 @@ function getTextValueOfFunctionProperty(
     .filter(statement => {
       const expr = (statement as ts.ExpressionStatement)
         .expression as ts.BinaryExpression;
+
+      const locals = Array.from(
+        (source as any).locals as [string, ts.Symbol][]
+      );
+      const hasOneLocalExport =
+        locals.filter(local => !!local[1].exports).length === 1;
```

Comment:
I wasn't _certain_ about this, but it did seem to resolve the one failing test I was dealing with.

Specifically: ['should be taken from stateless component `displayName` property (using default export) even if file name doesn't match'](https://github.com/styleguidist/react-docgen-typescript/pull/449/files#diff-8d9a49de4b110c8f3cc2e8788772996a65190fbdba297679966bf848f0ea2127R961)

Reply:
@pvasek @jesstelford curious on any input here

Reply:
This produced false-negatives in [my scenario](https://github.com/AMoo-Miki/oui/blob/b543b9aeadbfedff259294d358ecbbd4b35988a3/src/components/button/button.tsx). I presume because there is only one with `displayName`, it was ignoring the others.

Reply:
If you log `exports` there, what's the output?